.github/workflows/e2e-tests.yml

```diff
@@ -414,9 +414,11 @@ jobs:
           - "visualizations"
         include:
           - edition: oss
+            runner: ubuntu-22.04
             java-version: 11
             context: oss-subset
           - edition: ee
+            runner: ubuntu-latest
```

Comment:
We don't really care about the runner here. It only needs to run `echo` in bash.
But it needs to be different than `ubuntu-22.04` in order to create a new matrix group for the `slow` context, rather than iterating though the full `folder` list.